bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/SpanOptions.kt

```diff
@@ -93,16 +96,27 @@ public class SpanOptions private constructor(
         _parentContext,
         makeContext,
         isFirstClass,
-        _instrumentRendering,
+        _spanMetrics
+    )
+
+    @Deprecated(
+        message = "use spanMetrics.rendering",
+        replaceWith = ReplaceWith(
+            expression = "withMetrics(SpanMetrics(rendering = instrumentRendering))",
+            imports = ["com.bugsnag.android.performance.SpanMetrics"],
+        ),
     )
+    public fun withRenderingMetrics(instrumentRendering: Boolean): SpanOptions =
+        if (_spanMetrics != null) withMetrics(_spanMetrics)
```

Comment:
This ignores the `instrumentRendering` - it should be making a copy of the existing `SpanMetrics` with the `rendering` option modified to match `instrumentRendering`